src/vs/vscode.d.ts

```diff
@@ -6322,8 +6322,10 @@ declare module 'vscode' {
 		 * * *Note 1:* When executing an editor command not all types are allowed to
 		 * be passed as arguments. Allowed are the primitive types `string`, `boolean`,
 		 * `number`, `undefined`, and `null`, as well as [`Position`](#Position), [`Range`](#Range), [`Uri`](#Uri) and [`Location`](#Location).
-		 * * *Note 2:* There are no restrictions when executing commands that have been contributed
-		 * by extensions.
+		 * * *Note 2:* When executing commands that have been contributed by other
+		 * extensions, any JSON-serializable objects are additionally allowed.
+		 * * *Note 3:* There are no restrictions when executing commands that have
+		 * been contributed by the extension that is executing them.
```

Comment:
Your comment here https://github.com/microsoft/vscode/pull/80758#issue-316678175 is much better because that's how it is. Generally, there are no restrictions when calling a command that's contributed by another extension unless that command is used to activate the extension.

Reply:
You could also hint towards `Extension#isActive` and `Extension#activate` here